Source/cmake/FindExecinfo.cmake

```
@@ -0,0 +1,48 @@
+# Copyright (C) 2025 Igalia S.L.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+# 1.  Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+# 2.  Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDER AND ITS CONTRIBUTORS ``AS
+# IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR ITS
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+# OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+# WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+# OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+# ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+check_include_file("execinfo.h" HAVE_EXECINFO_H)
+check_function_exists(backtrace HAVE_BACKTRACE)
+
+if (NOT HAVE_BACKTRACE)
+    find_library(EXECINFO_LIBRARY execinfo)
+    if (EXECINFO_LIBRARY)
+        set(CMAKE_REQUIRED_LIBRARIES ${EXECINFO_LIBRARY})
+        check_function_exists(backtrace HAVE_BACKTRACE)
+        if (HAVE_BACKTRACE)
+            set(HAVE_LIBEXECINFO 1)
```

The convention for find-modules is to use `<name>_FOUND`, so this should be `Execinfo_FOUND`.


```suggestion
            set(Execinfo_FOUND TRUE)
```